modules/spring-config/src/main/java/org/mule/runtime/config/spring/dsl/model/extension/xml/MacroExpansionModuleModel.java

```diff
@@ -52,11 +52,11 @@
  */
 public class MacroExpansionModuleModel {
 
-  /**
-   * literal that represents the name of the global element for any given module. If the module's name is math, then the value of
-   * this field will name the global element as <math:config ../>
-   */
-  private static final String MODULE_CONFIG_GLOBAL_ELEMENT_NAME = "config";
+  //  /**
```

Comment:
I moved it to `org/mule/runtime/core/processor/chain/ModuleOperationMessageProcessorChainBuilder.java` so that there's one place where the default config is defined.